### PR TITLE
fix: devurlsHost ref in ingress template

### PR DESCRIPTION
**Pull request by stevek-pro**
Ingress template was not working correctly because there is neither a `.Values.devurls` nor a `.Values.devurls.host`. Correct values is in both cases `.Values.coderd.devurlsHost`.

### CODE REVIEW

File: templates/ingress.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ metadata:
     app.kubernetes.io/component: {{ include "coder.serviceName" . }}
   annotations: {{ toYaml .Values.ingress.annotations | nindent 4 }}
 spec:
+  ingressClassName: {{ .Values.ingress.classname | default "" | quote }}
```
Comment (deansheather) [inline templates/ingress.yaml:17]:
```suggestion
  ingressClassName: {{ .Values.ingress.className | default "" | quote }}
```

File: values.yaml [hunk 2]
```diff
@@ -269,6 +269,8 @@ coderd:
 ingress:
   # ingress.enable -- A boolean controlling whether to create an Ingress.
   enable: false
+  # Set ingress.classname -- The Classname of Ingress
+  classname: ""
```
Comment (deansheather) [inline values.yaml:273]:
```suggestion
  # ingress.className -- The ingressClassName to set on the Ingress.
  className: ""
```

**Review by deansheather — APPROVED** [review]

**Review by coadler — APPROVED** [review]

**Comment by deansheather** [comment]
Thank you for your contribution! @stevek-pro

**Comment by deansheather** [comment]
@lanzrein we plan on doing 1.29.2 tomorrow which will include it, and failing that it will be released in 1.30 RC1 next week

**Pull request merged**